pkgs/development/python-modules/qds_sdk/default.nix

```
@@ -0,0 +1,33 @@
+{ lib,
+ fetchPypi,
+ buildPythonPackage,
+ boto,
+ inflection,
+ pytest,
+ requests,
+ six,
+ urllib3 }:
+
+buildPythonPackage rec {
+  pname = "qds_sdk";
+  version = "1.12.0";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "qds_sdk";
+    sha256 = "113qrfz165sqcbndgdpvr4q7p08bdfw53sh1wrkvip8g2cngh6p3";
+  };
+
+  # No tests available
+  doCheck = true;
+
+  checkInputs = [ pytest ];
+  propagatedBuildInputs = [ requests boto six urllib3 inflection];
+
+  meta = with lib; {
+    description = "A Python module that provides the tools you need to authenticate with, and use the Qubole Data Service API";
+    homepage = https://github.com/qubole/qds-sdk-py;
```

```suggestion
    homepage = "https://github.com/qubole/qds-sdk-py";
```

Ref: https://github.com/NixOS/rfcs/pull/45

---

Done.